api/shared/nodenetworkconfigurationpolicy_utils_test.go

```diff
@@ -0,0 +1,75 @@
+package shared
+
```

Comment:
I'm wondering now if these tests are even executed, I fear they might not.
I believe you're missing a suite file, take a look at for example `pkg/names/names_suite_test.go`, this file is needed for each package to have the tests executed